src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs

```diff
@@ -92,5 +120,27 @@ public class AzureSqlElasticPoolModel
         /// Gets or sets the zone redundant option of the elastic pool.
         /// </summary>
         public bool? ZoneRedundant { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Sku of the elastic pool.
+        /// </summary>
+        public Sku Sku { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the per database settings of the elastic pool.
+        /// </summary>
+        public ElasticPoolPerDatabaseSettings PerDatabaseSettings { get; set; }
+
+        /// <summary>
+        /// Gets or sets the kind of the elastic pool. This is metadata used for the Azure
+        /// portal experience.
+        /// </summary>
+        public string Kind { get; set; }
```

Comment:
Better to remove this IMO. It is usually not useful information

Reply:
@jaredmoo ,yes, will remove.